sdk/ai/ai-projects/samples/v1-beta/typescript/src/agents/agentsBingCustomSearch.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to use agent operations with the Grounding with Bing Customer Search tool
```

Comment:
"with Groundings and Custom Bing Search Tool" (instead of "with the Grounding with Bing Customer Search tool". And elsewhere where since sentence appears.